flipkart/style.css

```diff
@@ -0,0 +1,340 @@
+@font-face {
+    font-family: myfont;
+    src: url(assets/roboto-regular.woff2);
+}
+*{
+    margin: 0;
+    padding: 0;
+}
+.nav-bar{
+    background-color: #2874f0;
+    display: flex;
+    align-items: center;
+    height: 52px;
+    justify-content: space-between;
+}
+.logo{
+    width: 23px;
+    height: 23px;
+    transform: translateY(1px);
+}
+.logo img{
+    width: 100%;
+    height: 100%;
+}
+.heading{
+    font-size: 16px;
+    font-family: myfont;
+    color: #ffff;
+    font-weight: 400;
+    height: 52px;
+    display: flex;
+    align-items: center;
+    padding: 0px 0px 0px 12px;
+    overflow: hidden;
+}
+.login{
+    font-size: 16px;
+    font-family: myfont;
+    color: #eaeaea;
+    width: 90px;
+    display: flex;
+    justify-content: center;
+    transform: translateY(-1px);
+}
+.nav-head{
+    display: flex;
+    align-items: center;
+}
+.nav-login{
+    display: flex;
+    align-items: center;
+    height: 52px;
+    position: absolute;
+    background-color: #2874f0;
+    right: 0;
+}
+.arrow{
+    width: 42px;
+    display: flex;
+    justify-content: center;
+}
+.cart{
+    display: flex;
+    width: 42px;
+    justify-content: center;
+}
+.search{
+    width: 42px;
+    display: flex;
+    justify-content: center;
+}
+.sort-filter{
+    display: flex;
+    width: 100%;
+    height: 48px;
+    position: sticky;
+    top: 0;
+    background-color: #ffff;
+    z-index: 1;
+}
+.sort, .filter{
+    width: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 9px;
+    height: 48px;
+}
+.sort-icon, .filter-icon{
+    width: 24px;
+    height: 24px;
+}
+.sort-icon img, .filter-icon img{
+    width: 100%;
+    height: 100%;
+}
+.sort-txt, .filter-txt{
+    font-size: 14px;
+    font-family:Roboto Medium,Roboto-Medium,Droid Sans,HelveticaNeue-Medium,Helvetica Neue Medium,sans-serif-medium;
+    transform: translateX(-0.5px);
```

Comment:
why translate? Try if there is any other way

Reply:
i think it's the simple way to make it pixel perfect